test/suite/test_cursor_bound08.py

```diff
@@ -87,7 +87,7 @@ def create_session_and_cursor_timestamp(self):
 
         if (self.evict):
             evict_cursor = self.session.open_cursor(uri, None, "debug=(release_evict)")
-            for i in range(0, 1000):
+            for i in range(100, 1000):
```

Comment:
Doesn't really matter, but you can use 1 here (it's 0 specifically that's reserved)... most tests now do `for i in range (1, nrows + 1)`.

Reply:
That's a good point, thanks for bringing that up. Have changed it as suggested!